apps/web/src/modules/job/list/pages/hooks/useJobListData/index.ts

```diff
@@ -0,0 +1,38 @@
+import { GetJobCardWithMaxPageDto } from '@modela/dtos'
+import { apiClient } from 'common/utils/api'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+
+const useJobListData = () => {
+  const router = useRouter()
+  const [job, setJob] = useState<GetJobCardWithMaxPageDto>()
+  const [hasMore, setHasMore] = useState(true)
+  var pageControl: number = 1
```

Comment:
should we user `useRef` here ?

Reply:
do you refer to only this `var pageControl: number = 1` or all 4 line

Reply:
only `var pageControl`